code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -153,6 +153,11 @@
 	var/armed = 0
 
 /obj/item/weapon/restraints/legcuffs/beartrap/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is sticking \his head in the [src.name]! It looks like \he's trying to commit suicide.</span>")
+	playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
+	return (BRUTELOSS)
+
+/obj/item/weapon/restraints/legcuffs/beartrap/suicide_act(mob/user)

```

Comment:
Grayshirt McDepressed is sticking his head in the beartrap! It looks like he's trying to commit suicide.